mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -626,36 +635,45 @@ ::mediapipe::Status GlAnimationOverlayCalculator::GlSetup() {
 
     // vertex position in threespace
     attribute vec4 position;
+    attribute vec3 normal;
 
     // texture coordinate for each vertex in normalized texture space (0..1)
     attribute mediump vec4 texture_coordinate;
 
     // texture coordinate for fragment shader (will be interpolated)
-    varying mediump vec2 sample_coordinate;
+    varying mediump vec3 sample_coordinate;
```

Comment:
We should revert the changes to sample_coordinate-- all our textures are 2D, so we don't want to be using 3D coordinates to index into them.  Same with below and in fragment shader.